### DISCUSSION TITLE
Table updates

### DISCUSSION THREAD

**Discussion by cmcclung-ANGENG** [ToolJet/ToolJet#7291] [Q&A] [answered]
Hi,

I want to periodically check for updates from a query for a table.

But when I have a script running updates in the background the table wipes all data and shows no data for a split second.

Is there a way to make this look seemless?

**Comment by withshubh** [comment, marked as answer]
Hey @cmcclung-ANGENG 👋 
Edit the table properties, and set the [`loading state`](https://docs.tooljet.com/docs/widgets/table/#loading-state-boolean) of the table. When the loading state is enabled, a loading indicator will be displayed on the table instead of an empty view. 

For example, if you have a PostgreSQL query named `postgres1` that is used to populate the table, go to the loading state property of the table - click on the `Fx` and set it to `{{queries.postgres1.isLoading}}`

**Comment by cmcclung-ANGENG** [comment]
Thanks that's looking good now :)
